### ISSUE TITLE
Bracket Spans

### ISSUE THREAD

**Issue by Erin-Cecele** [thegetty/quire-docs#698]
### Before proceeding, check to make sure there isn’t an existing issue for your suggestion or a similar one.

- [X] I have searched the existing issues and determined this is new suggestion.

### Idea being proposed

Document bracket spans:

Quire converts `[This is in small caps]{.smallcaps}` to `<span class="smallcaps">This is in small caps</span>`

Before we can announce it, though, we need to make a small change to Quire's core code to get it to work the way we want it to. @geealbers should another issue be opened in the main repo?

### What contribution category does your proposal fall under?

Propose/write new section

**Comment by geealbers** [comment]
We can now use `{ }` formatting to add classes AND ids to elements in markdown, as well as adding `target="_blank"` to links. So all three should be documented. Also, this syntax works on block elements too (headings, paragraphs, lists, tables, and blockquotes) so we should document that. 

There's detailed information in the plugin's repo (https://github.com/arve0/markdown-it-attrs), but it gets into the developer weeds, so I think we need our own version.
